test/HowTheTestDataWasMade.md

```diff
@@ -0,0 +1,29 @@
+
+# How the test data was made
+
+You should not ever need to do this. If you do, you will have to rewrite every assertion in every test. You don't want to do that unless it is absolutely necessary.
```

Comment:
This is why I think we should be making the test data before _each_ test, so that the person looking at the test (and possibly changing it) knows where it is created and what it looks like. 

This could be done with sequelize, eg in any given test file:

```js

describe('test', () => {
  beforeEach(() => {
     user = models.User.create({ <user details })
  })
  it('should get the user', () => {
     const result = request.get(`/artists/${user.id}`)
     expect(result.body.data.id).to.equal(user.id)
  })
})
```

Reply:
I agree. Once we get the first set of tests working, we should return to this topic.